deps/spidershim/include/v8.h

```diff
@@ -2064,6 +2077,14 @@ class V8_EXPORT FunctionTemplate : public Template {
                       Handle<Value> data = Handle<Value>());
   bool HasInstance(Handle<Value> object);
   void Inherit(Handle<FunctionTemplate> parent);
+
+  // XXXbz This is public because we call it from inside some static functions
+  // over in Function.  Maybe those should become Function members and then we
+  // can just friend Function and make this protected.
```

Comment:
Seems like this only gets called from `NativeFunctionCallback`.  I filed #145 to fix this.